src/__init__.py

```diff
@@ -0,0 +1,13 @@
+import datetime
```

Comment:
```suggestion
import datetime as dt
```

See my comment on this issue for explanation https://github.com/DailyPineapple/Finance_stuff/issues/3